src/credentials/vue/authForms/shared.vue

```diff
@@ -1,13 +1,30 @@
 <script lang="ts">
 import { CredentialsState } from './manageCredentials.vue'
-import { authForms } from './baseAuth.vue'
+import { CodeCatalystBuilderIdState, CodeWhispererBuilderIdState } from './manageBuilderId.vue'
+
+export const authForms = {
+    CREDENTIALS: 'CREDENTIALS',
+    BUILDER_ID_CODE_WHISPERER: 'BUILDER_ID_CODE_WHISPERER',
+    BUILDER_ID_CODE_CATALYST: 'BUILDER_ID_CODE_CATALYST',
```

Comment:
We're repeating ourselves a lot. We have the same id string and identifier in several places.

Reply:
I've de-deduplicated the repetition elsewhere so everything uses these properties only, instead of the string.

Reply:
edit: I've fixed this in a following PR and only use a `type` instead of this map